connector/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchStream.scala

```diff
@@ -316,6 +320,50 @@ private[kafka010] class KafkaMicroBatchStream(
     }
   }
 
+  private def assertEndOffsetForTriggerAvailableNow(
```

Comment:
Minor: `verifyEndOffsetFor..` might be better name than `assertEndOffset..`. `assert` has a specific meaning (often a code bug). This functionality sounds like more like verification.

Reply:
Yeah my initial intention was to guard against the code bug, but now it's more than the intention. Good point.